### DISCUSSION TITLE
Disable Hex?

### DISCUSSION THREAD

**Discussion by anaclumos** [facebook/docusaurus#8711] [General] [answered]
<img width="666" alt="Screenshot 2023-02-24 at 7 10 54 PM" src="https://user-images.githubusercontent.com/31657298/221333198-19b353f7-0f2c-4d51-a5ac-0127e00b9475.png">

I found that there is a unique alphanumeric id when building for production. Is there a way to remove this? Is this generated randomly for every build, or does it hashes the file's content? I want to improve my website's cache behavior.

**Comment by Josh-Cena** [comment, marked as answer]
Yes, the hash is based on the file content, and it's exactly used to improve cache, so the browser won't over-cache!
